kbc.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema":"https://json-schema.org/draft/2020-12/schema",
+  "type":"object",
+  "properties":{
+    "credentialSubject":{
+      "type":"object",
+      "properties":{
+        "id":{
+          "type":"string",
+          "pattern":"^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+$"
+        }
+      },
+      "required":[
+        "id"
+      ]
+    },
+    "issuer":{
+      "type":"string",
+      "pattern":"^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+$"
+    },
+    "issuanceDate":{
+      "type":"string",
+      "format":"date-time"
```

Comment:
noting that the regex for `date-time` is not quite conformant to xml date time as required by the spec

you can find a full regex for xml date time here https://github.com/w3c/vc-data-model/blob/main/schema/verifiable-credential/verifiable-credential-schema.json#L216